extensions/wikia/Wall/WallHooksHelper.class.php

```diff
@@ -686,8 +686,33 @@ static public function onArticleDelete( $article, &$user, &$reason, &$error ) {
 		return true;
 	}
 
-	static public function onRecentChangeSave( $recentChange ) {
-		wfProfileIn( __METHOD__ );
+	/**
+	 * Adjusting Recent Changes for Wall
+	 * Change RC db key to affect Watchlist behavior
+	 * @author TK-999 <https://github.com/TK-999>
+	 * @param RecentChange $rc
+	 * @return bool true because it is a hook
```

Comment:
I can see capital "Bool" here - https://github.com/Wikia/app/pull/9001/files#diff-49ba2b005c744aa46f66a4717763c10cR1114 and this one is all small letters - @garthwebb @kvas-damian Which is the PHPDoc convention we should follow?